packages/hop-node/src/gasboost/GasBoostTransaction.ts

```diff
@@ -564,10 +571,19 @@ class GasBoostTransaction extends EventEmitter implements providers.TransactionR
     this.maxPriorityFeePerGas = tx.maxPriorityFeePerGas!
     this.receipt = receipt
     this.emit(State.Confirmed, receipt)
-    this.logger.debug(`confirmed tx: ${tx.hash}, boostIndex: ${this.boostIndex}, nonce: ${this.nonce.toString()}, ${this.getGasFeeDataAsString()}`)
+    this.logger.debug(`confirmed tx: ${tx.hash}, boostIndex: ${this.boostIndex}, rebroadcastIndex: ${this.rebroadcastIndex}, nonce: ${this.nonce.toString()}, ${this.getGasFeeDataAsString()}`)
     this.watchForReorg()
   }
 
+  private async handleMaxRebroadcastIndexReached () {
+    this.maxRebroadcastIndexReached = true
+    this.clearInflightTxs()
+    this.emit(State.Error)
+    const warnMsg = `max rebroadcast index reached. cannot rebroadcast.`
+    this.notifier.warn(warnMsg, { channel: gasBoostWarnSlackChannel })
```

Comment:
suggestion: make this error logs instead of warns

```suggestion
   const errMsg = `max rebroadcast index reached. cannot rebroadcast.`
    this.notifier.error(errMsg, { channel: gasBoostErrorSlackChannel })
    this.logger.error(errMsg)
```